.github/workflows/integration-test.yml

```diff
@@ -5,6 +5,10 @@ on: [push, pull_request]
 env:
   TEST_BROWSER_HEADLESS: 1
   CI: 1
+  OPENSEARCH_DASHBOARDS_VERSION: 2.0
+  OPENSEARCH_CORE_VERSION: 2.0
+  OPENSEARCH_CORE_VERSION_QUALIFIER: ${{OPENSEARCH_CORE_VERSION}}.0
```

Comment:
Integration tests fail because these variables are not declared successfully. You can reference to [here](https://github.com/opensearch-project/security/blob/main/.github/workflows/ci.yml#L104-L112) and declare the env variables in the same way to make them available.